deployment/dockerfiles/notebooks/singleuser-notebook-tensorflow.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM dna/dev/pyspark-notebook:1.0-default
+FROM jupyter/pyspark-notebook:1.0-default
```

Comment:
confirm the image name

Reply:
change to jupyter/pyspark-notebook:latest like discussed

Reply:
@JanMigon  changes done in feature/vijaymhaske-patch-1